### PR TITLE
Fix/container/authentication

### LINKED ISSUE #311: Fix container query in `authentication`
**Provide the exact quote of the error or issue:**

We should modify the `_container` query to use breakpoints provided by `@yamada-ui/react`.

`contents\page-sections\authentication\with-image\index.tsx`
`contents\page-sections\authentication\with-title\index.tsx`
`contents\page-sections\authentication\forgot-password\index.tsx`
`contents\page-sections\authentication\authentication-form\index.tsx`

This issue will be automatically unassigned after two weeks have passed since it was assigned. Once unassigned, it may be picked up by someone else for work.

### PR DESCRIPTION
**Pull request by illionillion**
Closes #311

### CODE REVIEW

File: contents/page-sections/authentication/with-title/index.tsx
Comment (108yen) [inline contents/page-sections/authentication/with-title/index.tsx:1]: It's kind of lame, but is it a problem?

Comment (illionillion) [reply contents/page-sections/authentication/with-title/index.tsx:1]: No problem.

Comment (illionillion) [reply contents/page-sections/authentication/with-title/index.tsx:1]: How is it lame?

Comment (108yen) [reply contents/page-sections/authentication/with-title/index.tsx:1]: I thought it looked a little lame when `forgot password?` was shifted to the left.

**Review by 108yen — APPROVED** [review]

**Review by hirotomoyamada — APPROVED** [review]

**Pull request merged**